common/src/main/java/org/opensearch/ml/common/MLModel.java

```diff
@@ -44,13 +44,16 @@ public class MLModel implements ToXContentObject {
     public static final String MODEL_CONTENT_HASH_VALUE_FIELD = "model_content_hash_value";
     public static final String MODEL_CONFIG_FIELD = "model_config";
     public static final String CREATED_TIME_FIELD = "created_time";
+    public static final String LAST_UPDATE_TIME_FIELD = "last_update_time";
     public static final String LAST_UPLOADED_TIME_FIELD = "last_uploaded_time";
     public static final String LAST_LOADED_TIME_FIELD = "last_loaded_time";
     public static final String LAST_UNLOADED_TIME_FIELD = "last_unloaded_time";
 
     public static final String MODEL_ID_FIELD = "model_id";
     public static final String CHUNK_NUMBER_FIELD = "chunk_number";
     public static final String TOTAL_CHUNKS_FIELD = "total_chunks";
+    public static final String PLANNING_WORKER_NODE_COUNT_FIELD = "planning_worker_node_count";
```

Comment:
May be we can add some comment to explain what did we mean by `Planning worker`?

Reply:
Sure, will add some comments for this and also `currentWorkerNodeCount`